common-arcana.lic

```diff
@@ -503,14 +475,7 @@ module DRCA
     command = data['prep'] if data['prep']
     command = data['prep_type'] if data['prep_type']
 
-    if data['runestone_name']
-      if !prepare_to_cast_runestone?(data, settings)
-        return
-      end
-    end
-
-    return unless prepare?(data['abbrev'], data['mana'], data['symbiosis'], command, tattoo_tm = false, data['runestone_name'], data['runestone_tm'])
```

Comment:
I also just noticed that you have `tattoo_tm = false`. The original used `data['tattoo_tm']`. Why the change?

Reply:
I think I copied it from line 112 because I accidentally deleted it here on one of the versions.